data/brands/shop/convenience.json

```
@@ -5537,6 +5549,23 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "ローソンプラストークス",
+      "id": "lawsontoks-652b3e",
+      "locationSet": {"include": ["jp"]},
+      "matchNames": ["ローソン+トークス"],
+      "note": "LAWSON+toks has two types of stores: convenience store and kiosk.",
+      "tags": {
+        "brand": "LAWSON+toks",
+        "brand:en": "LAWSON+toks",
+        "brand:ja": "ローソンプラストークス",
+        "brand:wikidata": "Q115868119",
+        "name": "ローソンプラストークス",
```

Since "ローソン＋トークス" seems to be in common use [like this](https://www.rise.sc/shopguide/detail/?cd=000105), and the + is more like a symbol, so maybe this is a better one for the name tag?
```suggestion
      "note": "LAWSON+toks has two types of stores: convenience store and kiosk.",
      "tags": {
        "brand": "LAWSON+toks",
        "brand:en": "LAWSON+toks",
        "brand:ja": "ローソンプラストークス",
        "brand:wikidata": "Q115868119",
        "name": "ローソン+トークス",
```

---

I chose "ローソンプラストークス" because the official website used the notation "ローソンプラストークス". This suggestion may be better since there are also examples of the use of "ローソン+トークス".